File: Listas de Exercícios/Python/Python/Lista 1 - Casa/8.py
print("Média final do aluno\n")

print("--Valores decimais devem ser digitados com (.)!--")

N1=float(input("Nota - 1º Bimestre: "))
N2=float(input("Nota - 2º Bimestre: "))
N3=float(input("Nota - 3º Bimestre: "))
N4=float(input("Nota - 4º Bimestre: "))

M=(N1+N2+N3+N4)/4; M="{:.1f}".format(M)

print(f"\nMédia final: {M}\n")

File: 1th grade/Listas de Exercícios (Back-end [SPOLOGP])/C/Lista 1 - Casa/8.c
#include <stdio.h>

int main ()
{
    float N1, N2, N3, N4, M;
    
    printf("Média final do aluno\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)!--\n");
    
    printf("Nota - 1º Bimestre: "); scanf("%f", &N1);
    printf("Nota - 2º Bimestre: "); scanf("%f", &N2);
    printf("Nota - 3º Bimestre: "); scanf("%f", &N3);
    printf("Nota - 4º Bimestre: "); scanf("%f", &N4);
    
    M=(N1+N2+N3+N4)/4;
    
    printf("\nMédia final: %.1f\n", M);
}